mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/General.scala

```diff
@@ -35,23 +35,26 @@ trait General extends Command with CliConfig {
         "write intermediate output files to `out-dir`, default: false (overrides envvar WRITE_INTERMEDIATE)",
       useEnv = true)
   var features = opt[Seq[Feature]](default = Seq(),
-      description = """a comma-separated list of experimental features:
-        | - rows: enable row typing as explained in ADR-014
-        |""".stripMargin)
+      description = {
+        val featureDescriptions = Feature.all.map(f => s" - ${f.name}: ${f.description}")
+        ("a comma-separated list of experimental features:" :: featureDescriptions).mkString("\n")
```

Comment:
Wow, this even idents by the right amount! 🤩

The `-` could maybe be confusing?
 ```suggestion
        val featureDescriptions = Feature.all.map(f => s"  ${f.name}: ${f.description}")
        ("a comma-separated list of experimental features:" :: featureDescriptions).mkString("\n")
```

Reply:
Sounds good. Less noise is better :) thanks!